src/application/services/axisValInputHandler/axisValInputHandler.ts

```diff
@@ -0,0 +1,227 @@
+
+    if (originalValFormat === 'decimal') {
+      return String(parseFloat(originalVal) * 10)
+    }
+
+    if (
+      originalValFormat === 'scientificNotation' &&
+      originalVal.startsWith('1e')
+    ) {
+      return MathUtils.incrementExponent(originalVal)
+    }
+
+    throw new Error('Cannot multiply this value by 10')
+  }
+
+  private getInputValueDividedByTen(
+    axisSetId: number,
+    axisKey: AxisKey,
+  ): string {
+    const originalVal = this.inputValues[axisSetId][axisKey]
+    const originalValFormat = this.getInputValueFormat(originalVal)
+
+    if (originalValFormat === 'decimal') {
+      return String(parseFloat(originalVal) / 10)
+    }
+
+    if (
+      originalValFormat === 'scientificNotation' &&
+      originalVal.startsWith('1e')
+    ) {
+      return MathUtils.decrementExponent(originalVal)
+    }
+
+    throw new Error('Cannot multiply this value by 10')
+  }
+
+  handleMultiplyAxisValue(axisSetId: number, axisKey: AxisKey): void {
+    const multipliedVal = this.getInputValueMultipliedByTen(axisSetId, axisKey)
+
+    this.setInputValue(axisSetId, axisKey, multipliedVal)
+    this.setConvertedAxisValToAxisSet(axisSetId, axisKey)
+  }
+
+  handleDivideAxisValue(axisSetId: number, axisKey: AxisKey): void {
+    const multipliedVal = this.getInputValueDividedByTen(axisSetId, axisKey)
+
+    this.setInputValue(axisSetId, axisKey, multipliedVal)
+    this.setConvertedAxisValToAxisSet(axisSetId, axisKey)
+  }
+
+  private validateInputValues(axisSetId: number): void {
+    const axisSet = this.axisSetRepository.activeAxisSet
+    const { x1, x2, y1, y2 } = this.getAxisSetInputValues(axisSetId)
+
+    const isInvalidInputFormat = {
+      x1: this.getInputValueFormat(x1) === 'invalidFormat',
+      x2: this.getInputValueFormat(x2) === 'invalidFormat',
+      y1: this.getInputValueFormat(y1) === 'invalidFormat',
+      y2: this.getInputValueFormat(y2) === 'invalidFormat',
+    }
+
+    const xValuesAreSame = x1 !== '0' && x2 !== '0' && x1 === x2
+    const isXLogScaleAndZero = axisSet.xIsLogScale && (x1 === '0' || x2 === '0')
+    const yValuesAreSame = y1 !== '0' && y2 !== '0' && y1 === y2
+    const isYLogScaleAndZero = axisSet.yIsLogScale && (y1 === '0' || y2 === '0')
+
+    this.validationStatus[axisSetId] = {
+      isInvalidInputFormat,
+      xValuesAreSame,
+      isXLogScaleAndZero,
+      yValuesAreSame,
+      isYLogScaleAndZero,
+    }
+  }
+
+  private getValidationStatus(
+    axisSetId: number,
+  ): AxisValuesInputValidationStatus {
+    this.validateInputValues(axisSetId)
+    return (
+      this.validationStatus[axisSetId] || {
+        isInvalidInputFormat: {
+          x1: false,
+          x2: false,
+          y1: false,
+          y2: false,
+        },
+        xValuesAreSame: false,
+        isXLogScaleAndZero: false,
+        yValuesAreSame: false,
+        isYLogScaleAndZero: false,
+      }
+    )
+  }
+
+  getValidationMessage(axisSetId: number): string {
```

Comment:
When I hear `getValidationMessage`, I imagine a typical getter that just returns the class property as-is. So, I think using something like `generate` would be more appropriate.